src/sentry/static/sentry/app/views/releases/list/shared/releaseProgress.jsx

```diff
@@ -34,14 +36,13 @@ const STEPS = {
 };
 
 class ReleaseProgress extends AsyncComponent {
-  static contextTypes = {
-    organization: PropTypes.object,
-    project: PropTypes.object,
-    router: PropTypes.object,
+  static propTypes = {
+    organization: SentryTypes.Organization,
+    project: SentryTypes.Project,
```

Comment:
would this be a case where we want to add `isRequired`?

Reply:
Thanks - updated